### DISCUSSION TITLE
"Cancel" Button in Edit Package Description Broken

### DISCUSSION THREAD

**Discussion by andria-dev** [community/community#25489] [Actions]
<p>Hello all,</p>
<p>While I was looking at my GitHub package “react-spring-modal”, I happened across the “Edit package” dropdown and thus the “Edit description” button. Once I entered the package description editing page, I was unable to leave it by clicking the “Cancel” button. It appeared as if it had no effect at all.</p>
<p>While I can still just click the browser’s back button, this seems like a feature that was just maybe forgotten or broken. There aren’t any errors in the console.</p>
<p>I’m running the latest stable version of Edge Chromium on the latest release of Windows 10.</p>

**Comment by jcansdale** [comment, marked as answer]
<p>Hi <a class="mention" href="/chrisbrownie55">@ChrisBrownie55</a>,</p>
<p>Thanks for reporting. I’ve just checked and there is an internal issue open about this.</p>

**Comment by whitneyimura** [comment]
<p>:wave: This should be working now. Thank you for reporting!</p>
